test/lib/bamboo/email_test.exs

```diff
@@ -73,10 +23,176 @@ defmodule Bamboo.EmailTest do
     assert email.headers["Reply-To"] == "reply@foo.com"
   end
 
-  test "put_private/3 puts a key and value in the private attribute" do
-    email = new_email() |> put_private("foo", "bar")
+  describe "new_email/1" do
+    test "returns an Email struct" do
+      assert new_email() == %Bamboo.Email{
```

Comment:
I think this could be just `Email` because of the `alias Bamboo.Email` above

Reply:
Yup, good catch!